apps/pie-storybook/stories/pie-thumbnail.stories.ts

```diff
@@ -23,21 +27,49 @@ const thumbnailStoryMeta: ThumbnailStoryMeta = {
             control: 'select',
             options: variants,
             defaultValue: {
-                summary: defaultArgs.variant,
+                summary: defaultProps.variant,
             },
         },
         src: {
             description: 'Set the src attribute for the underlying image tag.',
             control: 'text',
             defaultValue: {
-                summary: defaultArgs.src,
+                summary: defaultProps.src,
             },
         },
         alt: {
             description: 'Set the alt attribute for the underlying image tag.',
             control: 'text',
             defaultValue: {
-                summary: defaultArgs.alt,
+                summary: defaultProps.alt,
+            },
+        },
+        disabled: {
+            description: 'Set the disabled attribute of the thumbnail.',
```

Comment:
images don't have a disabled attribute so i think we should say the following instead. 
```suggestion
            description: 'Applies the disabled styling of the thumbnail.',
```